public/specifications/monitoring/MONITORING.md

```diff
@@ -308,7 +322,7 @@ Some remarks:
 
 ### JSON Schema
 
-[status-schema.json](/specifications/monitoring/schemas/status-schema.json ':ignore')
+[status-schema.json](specifications/monitoring/schemas/status-schema.json ':ignore')
```

Comment:
It works on the gh-page 👍🏽 

However if we try to access the schema from [here](https://github.com/SRGSSR/pillarbox-documentation/blob/98-update-monitoring-specifications/public/specifications/monitoring/MONITORING.md) it doesn't work anymore. This relates to my comment https://github.com/SRGSSR/pillarbox-documentation/pull/96#discussion_r1741701929

But I guess we're going with gh-page.

Reply:
Another option is to inline the JSON directly into the page itself. Not sure if we can have boxes that can be expanded / collapsed so that we do not clutter the page unnecessarily, though, but you are the web experts here ;)

Reply:
> Not sure if we can have boxes that can be expanded / collapsed

There's a tag for that 🙂
```html
<details>
	<summary>Schema name goes here</summary>

	<!-- JSON schema goes here -->
</details>
```

Reply:
Links working with docsify kept for the moment. If somebody has a better idea it can always be submitted as a PR.